app/a-propos/page.tsx

```diff
@@ -5,7 +5,7 @@ import Content from './content.mdx';
 export default function APropos() {
   return <Content />;
 }
-
+export const revalidate = 3600;
```

Comment:
ℹ️ For other reviewer this constant is a convention in [Next](https://nextjs.org/docs/app/building-your-application/data-fetching/fetching-caching-and-revalidating#time-based-revalidation)